File: 백준/Bronze/2588. 곱셈/곱셈.java
import java.util.Scanner;

// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
// then press Enter. You can now see whitespace characters in your code.
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();

        int num = b%10;
        int num2 = b%100-b%10;
        int num3 = b%1000-b%100;

        System.out.println(a * num);
        System.out.println(a * num2/10);
        System.out.println(a * num3/100);

        System.out.println((a*num)+(a*num2)+(a*num3));

    }
}

File: 백준/Bronze/2588. 곱셈/곱셈.py
a = int(input())
b = int(input())
num = list(map(int,str(a)))
num2 = list(map(int,str(b)))

num[0] *= 100
num[1] *= 10

num2[0] *= 100
num2[1] *= 10

sum=0

add = 0
for i in num :
    add += (num2[2] * i)
print(add)
sum += add
add = 0
for i in num :
    add += (num2[1] * i)
print(int(add/10))
sum += add
add = 0
for i in num :
    add += (num2[0] * i)
print(int(add/100))
sum += add
add = 0
print(sum)